packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -300,7 +300,7 @@ export class FetchInstrumentation extends InstrumentationBase<
         const options = input instanceof Request ? input : init || {};
         const createdSpan = plugin._createSpan(url, options);
         if (!createdSpan) {
-          return original.apply(this, [url, options]);
+          return original.apply(this, options instanceof Request ? [options] : [url, options]);
```

Comment:
Why not? To avoid things like this in the future. If we don't create a span we should just pass whatever the user wants.
 
```suggestion
          return original.apply(this, arguments);
```